routing/unified_edges.go

```diff
@@ -40,9 +40,12 @@ func newNodeEdgeUnifier(sourceNode, toNode route.Vertex,
 }
 
 // addPolicy adds a single channel policy. Capacity may be zero if unknown
-// (light clients).
+// (light clients). We do not default to any payload size default function in
+// case no valid size function is supplied but request a graceful shutdown if
+// this happens.
```

Comment:
nit (if you push again): reword this a little more strongly? We expect a non-nil payload size function, and will request graceful shutdown if it is not provided as this indicates that edges are incorrectly specified.